Cargo.lock

```diff
@@ -271,6 +273,7 @@ dependencies = [
  "serde-wasm-bindgen",
  "serde_json",
  "thiserror",
+ "ureq",
```

Comment:
Seems like a wrong dependency?
Maybe `[dev-dependencies]` can be better.

Reply:
It's used in `build.rs`, so it's under `[build-dependencies]`.